test/conformance/demo-rp/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "nextjs-csr",
```

Comment:
we can update this

Reply:
okay updated in [07384b4](https://github.com/opengovsg/sgid-client/pull/60/commits/07384b48542897e25bfe12c4cc003551d3369b0b)